packages/sku/lib/packageManager.js

```diff
@@ -0,0 +1,67 @@
+const { cwd } = require('../lib/cwd');
+const { findRootSync } = require('@manypkg/find-root');
+
+const { sync: which } = require('which');
+
+/** @typedef {'yarn' | 'pnpm' | 'npm'} SupportedPackageManager */
+
+/** @type {Array<SupportedPackageManager>} */
+const supportedPackageManagers = ['yarn', 'pnpm', 'npm'];
+
+/**
+ * @param {SupportedPackageManager} commandName
+ * @returns {SupportedPackageManager | null}
+ */
+const detectPackageManagerCommand = (commandName) =>
+  which(commandName, { nothrow: true }) ? commandName : null;
+
+const detectPackageManager = () => {
+  return (
+    detectPackageManagerCommand('yarn') ||
+    detectPackageManagerCommand('pnpm') ||
+    detectPackageManagerCommand('npm') ||
+    'npm'
```

Comment:
Seems not worth calling `detectPackageManagerCommand('npm')`.

```suggestion
    'npm'
```